docs/en/sql-reference/functions/date-time-functions.md

```diff
@@ -211,7 +211,13 @@ Aliases: `DAYOFMONTH`, `DAY`.
 
 Converts a date or date with time to a UInt8 number containing the number of the day of the week (Monday is 1, and Sunday is 7).
 
-Alias: `DAYOFWEEK`.
+## DAYOFWEEK
+
+Converts a date or date with time to a UInt8 number containing the number of the day of the week (Sunday is 1, Monday is 2, and Saturday is 7), like `DAYOFWEEK` in MySQL or Spark. 
+
+## WEEKDAY
+
+Converts a date or date with time to a UInt8 number containing the number of the day of the week (Monday is 0, Tuesday is 2, and Sunday is 6), like `DAYOFWEEK` in MySQL or Spark. 
```

Comment:
From this doc it is not clear why `like DAYOFWEEK in MySQL or Spark` has different days mapping. We need to understand why `WEEKDAY` is so special that it has Monday=0. It just adds confusion for now

Reply:
To be honestly, I don't understand why `DAYOFWEEK` or `WEEKDAY` is different from `toDayOfWeek` in CH. 
This is the doc of spark and mysql. No explaination can be found here. 
- https://docs.databricks.com/sql/language-manual/functions/weekday.html
- https://docs.databricks.com/sql/language-manual/functions/dayofweek.html
- https://dev.mysql.com/doc/refman/8.0/en/date-and-time-functions.html#function_weekday
- https://dev.mysql.com/doc/refman/8.0/en/date-and-time-functions.html#function_dayofweek

Reply:
That means we should at least write

`DAYOFWEEK` like `DAYOFWEEK` in MySQL/Spark.
`WEEKDAY` like `WEEKDAY` in MySQL/Spark.

You have a typo in 220 row. Both are `DAYOFWEEK`